dds_web/config.py

```diff
@@ -64,6 +64,9 @@ class Config(object):
         "RATELIMIT_STORAGE_URI", "memory://"
     )  # Use in devel only! Use Redis or memcached in prod
 
+    # Redis
+    REDIS_URL = "redis://dds_redis"  # In production the URL also has a password
```

Comment:
https://github.com/ScilifelabDataCentre/dds_web/pull/1591#discussion_r1951121103
```suggestion
    RQ_DASHBOARD_REDIS_URL = "redis://dds_redis"  # In production the URL also has a password
```